src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -8,15 +8,16 @@
 	export let usdTotal = false;
 	export let summary = false;
 	export let actions = true;
-	export let more = false;
+	export let headerBack = false;
```

Comment:
Is that temporary or definitive?

If temporary, can we add a TODO.

If definitive, is it possible to have two backs buttons in the UI?
If not, can we have a single property?

```
export let back: "header" | "hero" | undefined
```

Reply:
i am not really sure it is temporary, not that i know of...

but I would prefer to have a single component, and a prop with the selection (last commit)